packages/utilities/src/JsUtils.ts

```diff
@@ -20,14 +22,19 @@ export class JsUtils {
      * @returns {boolean} - True if empty. False otherwise.
      */
     public static isObjEmpty(objToTest: object): boolean {
-        if (!objToTest) {
-            return true;
-        }
-        for (const prop in objToTest) {
-            if (Object.prototype.hasOwnProperty.call(objToTest, prop)) {
-                return false;
-            }
-        }
+        return Object.keys(objToTest ?? {}).length === 0;
+    }
+
+    // __________________________________________________________________________
+    /**
+     * Is the supplied string a URL.
+     *
+     * @param {string} urlString - The string to test.
+     *
+     * @returns {boolean} - True if it is a URL. False otherwise.
+     */
+    public static isUrl(urlString: string): boolean {
```

Comment:
After discussing this IRL, should `urlString` be wrapped in `path.posix.normalize`, so that drive letters in Windows paths don't get evaluated as protocols?

Reply:
Checking for `new URL("C:\\Users...").origin !== "null"` seems to work 😋